### DISCUSSION TITLE
Laravel 8 compatibility

### DISCUSSION THREAD

**Discussion by ariel-rajmaliuk** [ash-jc-allen/short-url#73] [Q&A] [answered]
Hi, 
 Do you know if it works with Laravel 8?

**Comment by ash-jc-allen** [comment, marked as answer]
Hey @Draidel!  As far as I'm aware, this should work no problem with Laravel 8. The test suite runs Laravel 8 with PHP 8, 7.4 and 7.3 and hasn't shown any signs of problems 😄
